packages/client/components/assignment-selections/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+  Button
+} from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import styled from 'styled-components'
+
+const DetailContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+`
```

Comment:
Seems like this component will be copy-pasted in several locations (there's already quite a few extremely similar ones). Is there a reason you prefer this over material-ui's `Grid` component?

Reply:
> Seems like this component will be copy-pasted in several locations (there's already quite a few extremely similar ones). Is there a reason you prefer this over material-ui's `Grid` component?

Good call. I didn't think about using `Grid` component. Looks like it does exactly what I need here. Will change to `Grid`